src/mam4xx/gas_phase_chemistry.hpp

```diff
@@ -41,7 +41,7 @@ void gas_phase_chemistry(
     const Real invariants[nfs], const int (&clsmap_4)[gas_pcnst],
     const int (&permute_4)[gas_pcnst], const Real het_rates[gas_pcnst],
     // out
-    Real (&qq)[gas_pcnst], Real (&vmr0)[gas_pcnst]) {
+    Real (&qq)[gas_pcnst]) {
```

Comment:
Thanks for changing the code to keep vmr0 in just one file.

Reply:
@overfelt : The vmr0 change is causing BFB tests to fail. I just tested it, and the tests pass when I change vmr0 treatment back to the original code.